README.md

```diff
@@ -8,26 +8,96 @@ The goal of this project is to make a feature complete Gothic game client compat
 [![Latest build](https://img.shields.io/github/release-pre/Try/opengothic?style=for-the-badge)](https://github.com/Try/opengothic/releases/latest)
 ![Screenshoot](scr0.png)
 
-### Work in progress
+#### Work in progress
 [![Build status](https://ci.appveyor.com/api/projects/status/github/Try/opengothic?svg=true)](https://ci.appveyor.com/project/Try/opengothic)
 
-The original game has been replicated fully; you can complete both the main quest and the addon.
-Check out the [bugtracker](https://github.com/Try/OpenGothic/issues) for a list of known issues.
+The original game has been replicated fully; you can complete both the main quest and the addon. OpenGothic runs reasonably well except on Low-End devices such as Raspberry-Pi. Check out the [bugtracker](https://github.com/Try/OpenGothic/issues) for a list of known issues.
```

Comment:
Not like we care about Rasbery-PI...
Probably better rephrase it somehow to indicate that build is aiming for mid-high end gpus, preferably with mesh-shader/rt-support